ci-operator/step-registry/ipi/conf/azure/confidential/trustedlaunch/ipi-conf-azure-confidential-trustedlaunch-ref.yaml

```diff
@@ -0,0 +1,20 @@
+ref:
+  as: ipi-conf-azure-confidential-trustedlaunch
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
+  commands: ipi-conf-azure-confidential-trustedlaunch-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:
+    - name: SECURITY_TYPE
```

Comment:
Based on we already indicate the step is for `trustedlaunch` type, so I think we does not need the ENV var any more?